npm/test/get-file-check-progress.test.ts

```diff
@@ -55,60 +49,66 @@ class GraphqlClientFailure {
     query: DocumentNode,
     variables: GetFileCheckProgressQueryVariables
   ) => Promise<FetchResult<GetFileCheckProgressQuery>> = async (_, __) => {
-    return Promise.reject("error")
+    throw new Error("error")
```

Comment:
Should this one still be a Promise, so that it simulates an async API call? Or an awaited Promise? I forget what combinations we tried yesterday, sorry. 🙂

Reply:
Hmm, yes, I think it should still be a Promise so that is simulates an async API call and so that it's consistent with the other client mocks; the result of the call is being awaited in the test

Reply:
Does this still need to be updated to be a Promise?

Reply:
Hmm, I'm not sure, I don't think so as the actual code doesn't return a promise with an error in it, it either returns a promise with the `GetFileCheckProgressQuery` result or an error. I would have suggested that maybe I could change the return type of the  function to an `Error`, but I don't think you can do that in Typescript.

Reply:
Ah, I think it actually returns a promise even though you don't declare one because it's an `async` function. So the code throws an error, but the actual return value is a rejected promise containing that error.